addons/mail/controllers/thread.py

```diff
@@ -72,14 +72,16 @@ def _get_allowed_message_post_params(self):
 
     @http.route("/mail/message/post", methods=["POST"], type="json", auth="public")
     @add_guest_to_context
-    def mail_message_post(self, thread_model, thread_id, post_data, context=None):
+    def mail_message_post(self, thread_model, thread_id, post_data, extra_data=None, context=None):
```

Comment:
Discussion about using `**kwargs` or listing the params instead of having `extra_data`